Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -6510,9 +6510,9 @@ RefPtr<CSSStyleValue> Parser::parse_easing_value(TokenStream<ComponentValue>& to
         // The first parameter specifies the number of intervals in the function. It must be a positive integer greater than 0
         // unless the second parameter is jump-none in which case it must be a positive integer greater than 1.
```

Comment:
I noticed the spec link and comment both are out of date, could you update them to reflect the current document?

Reply:
Sure

Reply:
You should use the editor's draft of the spec though :-)

Reply:
This one?
https://www.w3.org/TR/css-easing-2/#funcdef-step-easing-function-steps

Reply:
Go to the top and click the link under "Editor's draft", you'll end up here:
https://drafts.csswg.org/css-easing/

Reply:
Is it good now?

Reply:
Yep!